File: BEGINNER/Python/1040-Average3.py
n1, n2, n3, n4 = map(float, input().split())

average = ((n1*2) + (n2*3) + (n3*4) + n4)/10
print("Media: %.1f" %average)

if(average>=7):
    print("Aluno aprovado.")
elif(average<5):
    print("Aluno reprovado.")
elif(average>=5 and average<7):
    print("Aluno em exame.")
    
    exam = float(input())
    print("Nota do exame: %.1f" %exam)
    
    examAverage = (exam+average)/2
    if(examAverage>=5):
        print("Aluno aprovado.")
    elif(examAverage<5):
        print("Aluno reprovado.")
    print("Media final: %.1f" %examAverage)
        
    

File: BEGINNER/PHP/1040-Average3.php
<?php

$notes = explode(" ", readLine());

//PHP_ROUND_HALF_DOWN to round to the min value
$media = round((($notes[0]*2) + ($notes[1]*3) + ($notes[2]*4) + $notes[3])/10, 1, PHP_ROUND_HALF_DOWN);

//number_format to show at leat 1 decimal digit
echo "Media: " . number_format($media,1) . "\n";

if($media >= 0 && $media <=10){
    if($media >= 7){
        echo "Aluno aprovado.\n";
    }elseif($media < 5){
        echo "Aluno reprovado.\n";
    }else{
        echo "Aluno em exame.\n";
        $exam = readLine();
        echo "Nota do exame: $exam\n";
        $finalMedia = round(($exam + $media)/2, 1, PHP_ROUND_HALF_DOWN);;
        if($finalMedia >= 5){
            echo "Aluno aprovado.\n";
        }else{
            echo "Aluno reprovado.\n";
        }
        echo "Media final: " . number_format($finalMedia,1) . "\n";
    }
}

?>
